app/router.cjsx

```diff
@@ -187,15 +189,17 @@ module.exports =
       </Route>
       <Route path="collaborators" component={require './pages/lab/collaborators'} />
       <Route path="media" component={EditMediaPage} />
-      <Route path="workflow/:workflowID" component={require './pages/lab/workflow'} />
+      <Route path="workflows/:workflowID" component={require './pages/lab/workflow'} />
       <Route path="workflow/:workflowID/visualize" component={require './pages/lab/workflow-viewer'} />
       <Route path="workflow/:workflowID/visualise" component={require './pages/lab/workflow-viewer'} />
-      <Route path="subject-set/:subjectSetID" component={require './pages/lab/subject-set'} />
+      <Route path="subject-sets/:subjectSetID" component={require './pages/lab/subject-set'} />
```

Comment:
Same comment about a redirect. Also this route can be a nested child of the new `subject-sets` route